package.json

```diff
@@ -4,6 +4,7 @@
     "description": "A custom input component for displaying an array of color to pick from.",
     "main": "build/index.js",
     "scripts": {
+        "prepare": "yarn build",
```

Comment:
Thanks again for this and sorry for the delay, totally escaped my mind. Looks good, but could you run me through why this "prepare" script is needed?